.github/workflows/test-sim-merge.yml

```diff
@@ -32,35 +32,35 @@ jobs:
       # </common-build>
 
       # Install Geth merge interop
-      - uses: actions/setup-go@v2
-      - name: Clone Geth merge interop branch
-        run: git clone -b kintsugi-spec https://github.com/MariusVanDerWijden/go-ethereum.git
-      - name: Build Geth
-        run: cd go-ethereum && make
+      # - uses: actions/setup-go@v2
+      # - name: Clone Geth merge interop branch
+      #   run: git clone -b kintsugi-spec https://github.com/MariusVanDerWijden/go-ethereum.git && cd go-ethereum && git reset --hard b629e2560aae26059bf69ef19690385e078cbefd && git submodule update
+      # - name: Build Geth
+      #   run: cd go-ethereum && make
 
-      - name: Test Lodestar <> Geth interop
-        run: yarn test:sim:merge-interop
-        working-directory: packages/lodestar
-        env:
-          EL_BINARY_PATH: ../../go-ethereum/build/bin
-          EL_DIR: geth
-          EL_PORT: 8545
+      # - name: Test Lodestar <> Geth interop
+      #   run: yarn test:sim:merge-interop
+      #   working-directory: packages/lodestar
+      #   env:
+      #     EL_BINARY_DIR: ../../go-ethereum/build/bin
+      #     EL_SCRIPT_DIR: geth
+      #     EL_PORT: 8545
 
       # Install Nethermind merge interop
       - uses: actions/setup-dotnet@v1
         with:
           dotnet-version: '5.0.x'
       - name: Clone Nethermind merge interop branch
-        run: git clone -b themerge_kintsugi https://github.com/NethermindEth/nethermind --recursive
+        run: git clone -b themerge_kintsugi https://github.com/NethermindEth/nethermind --recursive && cd nethermind && git reset --hard 8e54f87ce2e36e20a73b73a0e470816b1edc9b92 && git submodule update
```

Comment:
Isn't it possible to checkout a specific commit directly?

Reply:
no, i searched, couldnt find a way to do it!